src/main/kotlin/com/ampnet/headlesscmsservice/service/pojo/ContentResponse.kt

```diff
@@ -0,0 +1,24 @@
+package com.ampnet.headlesscmsservice.service.pojo
+
+import com.ampnet.headlesscmsservice.persistence.model.Content
+import java.util.UUID
+
+data class ContentResponse(
+    val uuid: UUID,
```

Comment:
Why do we need UUID in the response if we don't access anything with it?
The only way to access/update the content is via coop-key-lang triple.